ViewGeneratorCore/tools/webpack/webpack.config.ts

```diff
@@ -0,0 +1,133 @@
+﻿import Glob from "glob";
+import Path from "path";
+import Webpack from "webpack";
+import ManifestPlugin, { FileDescriptor } from "webpack-manifest-plugin";
+
+const entryMap: {
+    [entry: string]: string
+} = {};
+
+const outputMap: {
+    [entry: string]: string
+} = {};
+
+/** 
+ *  Build entry and output mappings for webpack config
+ * */
+function getConfiguration(input: string, output: string): void {
+    Glob.sync(input).forEach(f => {
+
+        // Exclude node_modules files
+        if (!f.includes("node_modules")) {
+
+            let entryName: string = Path.parse(f).name.replace(".view", "");
```

Comment:
replace(".view.", ".") ...isn't this more safe?

Reply:
I just want to remove the suffix here (e.g. C:\Program Files\Example\ExampleView\Example.view.js, just want the Example part. I will use substr to lastIndexOf "." instead